tests/fixtures/token_capturing.rs

```diff
@@ -2,10 +2,7 @@ use crate::harness::suites::html5lib_tests::{
     get_test_cases, TestCase, TestToken, TestTokenList,
 };
 use crate::harness::{TestFixture, Input};
-use lol_html::{
-    LocalNameHash, TokenCaptureFlags, LocalName, Token, StartTagHandlingResult, TransformController,
-    TransformStream, Namespace, TransformStreamSettings, MemoryLimiter
-};
+use lol_html::{LocalNameHash, TokenCaptureFlags, LocalName, Token, StartTagHandlingResult, TransformController, TransformStream, Namespace, TransformStreamSettings, MemoryLimiter, SharedEncoding};
```

Comment:
This needs cargo fmt. Let's add `cargo fmt -- --check` to CI

Reply:
This is formatted by `cargo fmt`.

> Let's add cargo fmt -- --check to CI

Will open a PR for that.